examples/cfd/test_01_convection.ipynb

```diff
@@ -0,0 +1,506 @@
DzOL/sbO46TWsiV9RoII83E\ngTDH5Pqv/BXgMelgEe5c3mz3mji01k/0u89sH4Akx7+WZMWBbe2fz8qa+KHXNKxaJXgYQVMiNjb8\nJ6sQE0jQJUTUkCuz9pmQq9frqFaroW94UfY/Du5jCmvLUSRZn1/TNDEYDGL9jeNSNsc2LmETB3iR\n5+WYTH4+e37MsYKdcRNukla4ZtWyxI+ndg5gvT5fldvuexdJnOusoaH5iyzdUnGuHX+EmBulYqHX\nrUMRoDI4K+K8n9J8CClzfp7fOWq329SDLgIk6FLAL9JFC6KkQo5R9P4nge2zyEKOJ4vzy/7GLHxe\nq9UWftMu23USB94x4cVeUCParDh3btKSYcbZ8vjTXnQk2wIm1iokCUF5dI4tJXLplpUs3jdhH0JE\nHnuWBmoqHA0SdBlRlCDi86dY2C2JW1NGQQfs5weKLOSA9Mm8fNPnRRa0uD8xl/U6SYuXY3LNZ++E\nWgP0bhXV9eihQb5lyYVzawAneqIIoGfPcQubAxw7MukZl1WFK0/csOtz5zegqNkK3Zff+wH8j19+\nT6bbLIo83rNZtFURReiRQ5cOEnQpKNKhcyfCa5qWKuxWpoWad+RkWRa+v1oa3NM78mz67L6ey3RN\nFEkcMQfsibgs2PvTnDk3yS1yHAlHDncTbSqKS3euM9nvwajqGXZ97rx/jlOSwggAsAxlacKui34v\nhbVVKWLsWRhMbLohhy4ay7kKFsCiqlzzqmgsw+LNTzxQVRWapsFxnFKIubjnd1XGkZUZtTYrNIyd\nGqRGcCHE7lYTUsYOFs+5rfVAUccqXOMQ5tK5w66WKady6dyFEctE0Q4Y2wdRx56RQ5cO8VdCgVlk\nKGoRrSlEFXRuIcccufF4DMMwit69yMTtmUdCTlyu+eydsMeT9hNeDp27wnV3a75QAsBMuDUrzm1N\ncvEOHoqXF8dcOq+CiO2ez/7vcfq5A5Ar8yIuTR6dUrFgGWKmUCwbeY89S0u328Xx48cz3+6yQYIu\nY7KuDGRd/0ejSSVaXj3GRHToooyuEm2f/Qj7e4kwjkzEa0BUnIEKKPvnKsidG7e9p0jkzYWtNRw8\n3At8zng8+37yq2514xd2zQOzX8XVf/PX+M7NJxfyennhF04UmazGnkVdr4IcuoMHD2ZyTMsMCbqM\n4HOOshBb7vFNjUYDqqouJH+qaNxCzm90lQjhizh4nd+yVOgyROy3WDTGjv94L32rDlQSvq8yuLwv\nbE3y3sKEHcMwFM8WK8NeDdVmOjc8LI9udKYB7OXQqYeHqV5LRES5v2ZB3LFnUcO2fusn5dBFgwRd\nCvLoRecWcs1mM1chxxBhoY4q5BgiidAwvG5cLIxcBiFH7HP1f/t/Ij1P38pv4H1cLmytobmZj0hq\nb0/CsbYhT8OuUfPoRmddkyckB3AkmFt1SJsTB1CqWijHuzycsn0IjUua/DxFUXzv551OBwcOHFjU\nYZQWEnQZwkRRkoWZFzNMyKWZwxmXIsWRl4iNcuxlEnTA/o0tLIxcFGU7n0Xi7IVbJZ/kfT8xF1QQ\nkaZnm7vBMez5/eq365FE3bCrob4+32w4a+bEnJsLVeDg8kyKWNX3Vlh+Hj/2DAAGgwFkWcZjjz2G\nf/zHf8RVV12FwWBAkyIiUK6AvmC4P23Jshz7TctCbp1OB7quo9lsYn19faFiDihmMWfiptPpYDwe\nF3bsi8K27enfudVqCSXmiJKR4K3qmBJ2zweLKHMY/b03GFWn7lzg67rE5ehsI1zMMS7szz29+m/+\nOvK+icqyO3RxYCKvUqmgVqtB0ya5ps1mc9ow/cyZM7jzzjvxla98BZdccgl++qd/Gu9617vwoQ99\nCA888AAsK7ylza233oqjR4/immuuCXzeN77xDaiqik9/+tOZHF8RkKDLkDiiiBdyhmGg1WoVKmYW\nKeiyEnJlcJSY+9jv9+E4zvRYyyDkaPGZ59qPfjj0OdJ5/5y6MjLsTY5H76e/N0UWchyOTqkIqwDL\nn2Nh26uvvhp/+qd/is997nO45ppr8Oijj+K//Jf/gssvvxzf+ta38Ad/8AeR7lG33HIL7rvvvsDn\n2LaN3//938frXve6rA6nEMRfVQQmSQ6duwVHWJ7YoliEOMo6P1BkQefV+JnlyxHlxW54OwKswlU6\nX4Ndi5iLWtBYrN3zDbSeN5ircOVJGnYNyqPTz+6FoWO+3eW2CnvTBHzOfVlY9tnIafE7P+wef+TI\nEbzmNa/Ba17zmljbvf7663Hq1KnA5/zlX/4l3vKWt+Ab3/hGrG2LRvFKYokIEhh8NSPfS00U8hRH\neRZ6iCjo3KPYKpXKtFGwyES5BkQW0YtGbauw1iYiwxmokAfzTpKTtMI1Z3bPN1BZS5efpl/QgFo0\nkTUVc0FImISS9wojHBmQlqigmt436chLDD/77LP4zGc+gy9+8Yv4+te/nstrLAoKuWaI12LnOA6G\nwyHa7TYsy8La2ppwYg7IZ6FmQq7b7WI0GqHRaGBtbS2zPnpsG6LcKE3TRLfbRb/fR61Ww/r6+nQc\nG30yXx4cZQHX2wIuF3M7uD/esDv/+FzYdRweDjXP1iF5FGlExVEdyG0VGCilz6Oj+4A/fg6druu5\nRjbe+9734o477hBuPUmCWKqiZHiFXFnrD9u2p9WMZWhLkaWgW8RUC0Ccm6NpmhgOh7AsC5qmTRN6\n3ZT5RkHs5c81Js4cgKk7B8DTnSuMhOLJqyBi2NUmjllCzCjO3ApA7/1wiupB981vfhO/+qu/Csdx\ncP78edx7772oVCq44YYbcnvNvCBBlxJeCMmyDF3XMRgMMB6PSyHk3KTJ81iUkOPJsplzXCzLwnA4\nhGEYqNfraLVavvshivgMgr+WHceZNgVlnd7LcAx5IlmA0kv+Xs5rhutcyxKv55jezzG3NaiHAnLl\nfMScfsHf3eP
```

Comment:
I'm not sure what you mean here (notebook diff on github is still horrible). If it's the repeated initialization of `u` in cell `[3]`, that's simply convenience, since it allows re-running the same cell over and over again.